src/components/Missions/Mission.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions, missionState } from '../../features/mission/missionSlice';
+import Join from './JoinBtn';
+import Membership from './Membership';
+import './mission.style.css';
+
+function Missions() {
+  const mission = useSelector(missionState);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, []);
+
+  return (
+    <div className="container">
+      {mission.length ? (
+        <div>
+          <table>
+            <thead>
+              <tr>
+                <td>Mission</td>
+                <td>description</td>
+                <td>Status</td>
+              </tr>
+            </thead>
+            <tbody>
+              {mission.map((mission) => (
+                <tr key={mission.mission_id}>
+                  <td>{mission.mission_name}</td>
+                  <td>{mission.description}</td>
+                  <td>
+                    <Membership reserved={!!mission.reserved} />
+                  </td>
+                  <td>
+                    <Join
+                      reserved={!!mission.reserved}
+                      id={mission.mission_id}
+                    />
```

Comment:
- [ ] When I navigate from "Mission" tab to "Rocket" tab the reserved mission is canceled. Please fix this to make your project more perfect.

Reply:
Fixed by Dear @Ukaypromise